File: src/test/resources/AnnotateClassMethodsIntention/android/before.template.after.java
import android.support.annotation.VisibleForTesting;

class Foo {

    @VisibleForTesting
    private void annotatedFoo() {}

    public void publicFoo() {}

    @VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)
    public void foo() {}

    @VisibleForTesting(otherwise = VisibleForTesting.PACKAGE_PRIVATE)
    public void bar() {}

    @VisibleForTesting
    public void baz() {}
}

File: src/test/resources/AnnotateKtClassMethodsIntention/android/before.template.after.kt
import android.support.annotation.VisibleForTesting

class Foo {

    @VisibleForTesting
    fun annotatedFoo() {}

    fun publicFoo() {}

    @VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)
    fun foo() {}

    @VisibleForTesting(otherwise = VisibleForTesting.PACKAGE_PRIVATE)
    fun bar() {}

    @VisibleForTesting
    fun baz() {}
}